### ISSUE TITLE
logMqtt, no loggin

### ISSUE THREAD

**Issue by zackdvd** [arachnetech/homebridge-mqttthing#26]
Hi,
i set this to true, but still get no logging, even with Debug mode of homebridge.
Bug or my fault?

"logMqtt": "true",

Thank you!

**Comment by arachnetech** [comment]
Strictly I think it should be true rather than “true”, but I would expect both to work. 

Logging output is to the console when running homebridge interactively. I’m not sure whether this makes it to the log files when running nas a service.

**Comment by zackdvd** [comment]
Sorry, absolutly my fault. logMqtt works as expected.

The problem I encountered: 
First I used a plain IP address as url (yes, thats not a full url...) and got an Exception (something very unclear, took me some time to figure out, maybe add a log entry?). 
Then I added :<port> and I got no error, but also no Mqtt logging. 
Again, after some tests I added http:// and voila, all works really pretty!

Two proposals: Link your example file in the Readme file and give a hint, that url needs http and a port.

Thank you for this perfect plugin!!
